python/graphstorm/gconstruct/transform.py

```diff
@@ -164,6 +164,7 @@ def pre_process(self, feats):
             # It will load all data into main memory.
             feats = feats.to_numpy()
 
+        feats = feats[feats is not None]
```

Comment:
the idea here is to have `None` as element values, but we don't create a value for `None` in the dict?

Reply:
It should be feats = feats[feats != None]